pages/app/profile.jsx

```diff
@@ -5,9 +5,15 @@ import { FaWallet } from 'react-icons/fa';
 import { FiSettings } from 'react-icons/fi';
 import { TfiExport } from 'react-icons/tfi';
 import { BsBoxArrowRight } from 'react-icons/bs';
+import Link from "next/link";
 
 
 export default function profile() {
+
+  const handleLogout = () => {
+    localStorage.removeItem("ex_pensify_token");
+    console.log('User Logged Out');
```

Comment:
use next router push method to push to the new page once the token has been removed from the localStorage